repeating_dates.module

```diff
@@ -98,7 +99,19 @@ function repeating_dates_field_widget_settings_form($field, $instance) {
         'date' => t('Date'),
       ),
       '#default_value' => isset($settings['default_stop_mode']) ? $settings['default_stop_mode'] : 'count',
-      );
+    );
+    $form['default_count'] = array(
+      '#type' => 'number',
+      '#title' => t('Default count'),
+      '#default_value' => $settings['default_count'],
+      '#min' => 1,
+      '#max' => REPEATING_DATES_MAX_VALUES,
+      '#states' => array(
+        'visible' => array(
```

Comment:
Switching visibility here seems problematic. In the (front end) widget, people can always toggle, so a good value should be in place then.